lib/Form/DisplayFormInWPContent.php

```diff
@@ -187,10 +187,10 @@ private function shouldDisplayFormType(FormEntity $form, string $formType): bool
       return false;
     }
 
-    if ($this->wp->isSingular('post')) {
+    if ($this->wp->isSingular('post') || $this->wp->isSingular('product')) {
```

Comment:
With this change when a user selects `display on all posts` the form will be displayed also on all product pages.
I'm not sure if that an intended outcome. We put products into the selection of posts so I guess it makes sense, on the other hand, I'm not sure if users understand that + it will change the behaviour of already created forms.

Alternatively, we could display forms on product pages only if they are listed in select.
cc @veljkho